cellmlmanip/model.py

```diff
@@ -387,6 +387,20 @@ def get_derived_quantities(self):
                               and node.get('variable_type', '') not in ('state', 'free', 'parameter')]
         return sorted(derived_quantities, key=lambda var: var.order_added)
 
+    def get_display_name(self, var, ontology=None):
+        """Return a display name for the given variable.
+
+        Looks for annotation in the ontology first (if given), then cmeta:id if present, or the name attribute if not
+        :param var: the variable for which to get the display name.
+        :param ontology: the namespace prefix for the ontology used (defaults to not considering annotations).
+
+        :return: the local name from the ontology (if such an annotation exists)
+                 or the cmeta_id (if present) or the variable's name
+        """
+        if ontology is not None and self.has_ontology_annotation(var, ontology):
```

Comment:
Why the `ontology is not None` here? I want it to return from _any_ ontology if I don't specify a specific one?

Reply:
@jonc125 is this different from what you had in mind?

Reply:
I just applied Jonathan's comment without thinking. I reverted it now!

Reply:
Ah, sorry, I thought the request before was to do this only if an ontology was specified! Just me misunderstanding the request.